doc/sphinx/source/examples/Tutorials/Two_Way_Wave_Equation/Two_Way_Wave_Equation.md

```diff
@@ -0,0 +1,219 @@
+| `walltime_md` | Wall-clock time (seconds) |
+| `flops_md` | Estimated floating-point operation count |
+
+### Notes
+This uses the same CTCS time integration scheme as FD. The **order of accuracy** can be increased by adjusting the `k` parameter (e.g. `k=2`, `k=4`, etc.). Mimetic operators preserve **discrete conservation laws**, often improving numerical stability and physical fidelity. If you change the number of cells, be sure to change the time step, to conform to the CFL condition.
+
+---
+
+## 3. `comparison_two_way_wave_md_vs_fd`
+
+**Purpose:** Compare **mimetic** and **finite difference** results for the same PDE setup.
+
+This script runs both solvers and measures differences in:
+- Numerical accuracy
+- Computational cost
+- Wall-clock time
+- Error convergence rate
+
+Assorted plots are generated showing differences between the two methods.
+
+
+## Grid Comparison
+
+The key difference between FD and MD methods lies in the spatial grid. Here is a slightly more involved explanation to help the user understand the difference in grids.
+
+### Finite Difference Grid (Uniform)
+
+Each cell is the same width Δx between **A** and **B**, with points at cell boundaries (0,1,2,...N-1,N,N+1):
+```
+A                                                                 B
+ <-----dx-----> <-----dx----->  ...  <-----dx-----> <-----dx----->
+|----cell 1----|----cell 2----| ... |----cell N-1--|----cell N----|
+0              1              2 ... N-1             N             N+1
+```
+
+For example, from 0 to 1 with 5 cells has six values (x0,x1,x2,x3,x4,x5) each 0.2 away from each other:
+```
+0.0   0.2   0.4   0.6   0.8   1.0
+ o-----o-----o-----o-----o-----o
+ x0    x1    x2    x3    x4    x5
+```
+### Mimetic Difference Grid (Staggered)
+
+Mimetic operators use a staggered grid—half-step offsets at boundaries improve conservation and symmetry. Here, note that **cell 1** and **cell N** are not the sam size as the internal cells, and that there are now **N+2** points.
+```
+A                                                         B
+ <--dx/2--> <-----dx----->  ...  <-----dx-----> <--dx/2-->
+|--cell 1--|----cell 2----| ... |----cell N-1--|--cell N--|
+0          1              2 ... N             N+1        N+2
+```
+
+For the same interval (0–1) with 5 cells, a staggered grid will have smaller dt/s at the beginning and end, and also one mroe point (x0,x1,x2,x3,x4,x5,x6):
+```
+0.0   0.1   0.3   0.5   0.7   0.9   1.0
+ o-----o-----o-----o-----o-----o-----o
+ x0    x1    x2    x3    x4    x5    x6
+```
+
+This staggered layout provides **better discrete analogs** of differential operators, leading to improved conservation and often reduced numerical dispersion.
+
+---
+
+## Convergence and Accuracy
+
+Both FD and MD methods use a **second-order accurate** CTCS scheme in time.  
+The mimetic method, however, allows **higher-order spatial accuracy** by increasing the mimetic operator order `k`.
```

Comment:
I would add:
```suggestion
The mimetic method, however, allows **higher-order spatial accuracy** by increasing the mimetic operator order `k` without having to reconstruct the matrix.
```
?
Of something of that sort. FD can be high-order, too. You would just have to redefine your matrix.